### ISSUE TITLE
Suggestion for a Library Picture at Starting Main Screen.

### ISSUE THREAD

**Issue by blessedbaptist** [Librarika/Issues#419]
HI, Rayhan,

Great thanks for great library!!
If you provide a space for a library picture at starting main screen, I think it would be great.

Blessings,
Yohan

**Comment by rayhan** [comment]
Thanks for this nice suggestion. We will consider it into our feature list.

**Comment by vsolas** [comment]
I agree with this suggestion. It would add a level of personalization for each library, even if it was only available on paid tiers.
